extensions/usage-metrics-dashboard/tests/testthat/setup.R

```diff
@@ -0,0 +1,5 @@
+library(testthat)
+
+for (f in list.files("../../R", pattern = "\\.R$", full.names = TRUE)) {
+  source(f)
+}
```

Comment:
Is this necessary? I thought that testthat would automatically load those files given the directory structure you have here? If not: is there something else we need to do to make this slightly more package-like maybe and make the testthat machinery work for us?

Reply:
Thank you for catching that. I needed to change the `testthat` invocation in the workflow `.yml` file.